acrobat/scripts/contentSecurityPolicy/dev.js

```diff
@@ -10,6 +10,7 @@ const connectSrc = [
   '14257-chimera-stage.adobeioruntime.net',
   '14257-chimera.adobeioruntime.net',
   '*.adobe.com',
+  'stage.adobeccstatic.com',
```

Comment:
should these be stage.adobeccstatic.com or dev.adobeccstatic.com?

Reply:
Our dev env (hlx.page and hlx.live) loads the FEDS stage scripts.